talkative-tangs/Bangazon-LLC-2
# Bn planning phase

**Initial Setup and Database Migration**
No Issue Ticket

**Description of Proposed Changes:**
Database models in models.py created without soft delete with all models utilizing a deletedDate attribute
 
**Steps to Test**
```
git fetch --all
git checkout bn-planning-phase
```
In your terminal, navigate to the directory that contains the env directory
activate the environment
`source env/bin/activate
`
once inside the environment you can run the migrations
```
python manage.py migrate
python manage.py makemigrations website
python manage.py migrate
```

Open DB Browser and open db.sqlite3
Confirm all tables are created with appropriate attributes

Impacted Areas in Application
List general components of the application that this PR will affect:


To Review:
@laboyd001 
@ousamasama 
@CurtainUp
